eslint-plugin-amsterdam/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "eslint-plugin-amsterdam",
+  "description": "eslint plugin shared by Amsterdam",
+  "version": "0.0.1",
+  "author": "Jasper Poppe",
```

Comment:
Can we use `Datapunt Amsterdam` here? maintainers names can be put in a MAINTAINERS.md file if needed

Reply:
I removed the author field for now, we could add add Datapunt or something else if this plugin will ever be published.